### PR TITLE
Merge latest fbsync with master 2020/07/07

### PR DESCRIPTION
**Pull request by Nayef211**
- Resolved all conflicts locally

### CODE REVIEW

File: torchtext/experimental/datasets/text_classification.py [hunk 1]
```diff
@@ -16,6 +16,10 @@ def _build_vocab(data, transforms):
         tok_list.append(transforms(txt))
     return build_vocab_from_iterator(tok_list)
 
+def vocab_func(vocab):
```
Comment (zhangguanheng66) [inline torchtext/experimental/datasets/text_classification.py:19]: Not sure why this appears here. But we import `vocab_func` from `torchtext.experimental.functional`

Comment (Nayef211) [reply torchtext/experimental/datasets/text_classification.py]: Should be fixed now!

**Comment by Nayef211** [comment]
> Not sure why those changes come from fbcode. We can merge them but those changes are not necessary.

I think it was just styling changes? It seems like the linter has different style expectations for python in `fbcode` vs `torchtext`

**Comment by Nayef211** [comment]
Closing this PR as per my conversation with @zhangguanheng66

**Pull request closed without merging**